src/components/lyne-tab-group/styles/lyne-tab-group.component.scss

```diff
@@ -0,0 +1,132 @@
+    @include title;
+    line-height: var(--typo-line-height-body-text);
+    font-size: calc(var(--typo-scale-0-75x) / var(--typo-scale-default) * 1rem);
+    display: inline-block;
+    vertical-align: middle;
+    padding: calc(var(--spacing-fixed-2x) / var(--typo-scale-default) * 1rem) calc(var(--spacing-fixed-5x) / var(--typo-scale-default) * 1rem);
+    border: calc(var(--border-width-1x) / var(--typo-scale-default) * 1rem) solid var(--color-cloud-default);
+    border-radius: calc(var(--spacing-fixed-5x) / var(--typo-scale-default) * 1rem);
+    color: var(--color-black-default);
+    background-color: transparent;
+    -webkit-tap-highlight-color: transparent;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    transition: color var(--animation-form-element-state-duration) var(--animation-form-element-state-easing),
+                border var(--animation-form-element-state-duration) var(--animation-form-element-state-easing),
+                background-color var(--animation-form-element-state-duration) var(--animation-form-element-state-easing);
+    @include mq($from: medium) {
+      font-size: calc(var(--typo-scale-0-875x) / var(--typo-scale-default) * 1rem);
+    }
+  }
+
+  ::slotted(h1:hover),
+  ::slotted(h2:hover),
+  ::slotted(h3:hover),
+  ::slotted(h4:hover),
+  ::slotted(h5:hover),
+  ::slotted(h6:hover) {
+    cursor: pointer;
+    background-color: var(--color-milk-default);
+  }
+
+  ::slotted(h1:focus-visible),
+  ::slotted(h2:focus-visible),
+  ::slotted(h3:focus-visible),
+  ::slotted(h4:focus-visible),
+  ::slotted(h5:focus-visible),
+  ::slotted(h6:focus-visible) {
+    outline: calc(var(--border-width-1x) / var(--typo-scale-default) * 1rem) solid var(--color-black-default);
+    outline-offset: calc((var(--border-width-1x) + var(--border-width-2x)) / var(--typo-scale-default) * 1rem);
+  }
+
+  ::slotted(h1:active),
+  ::slotted(h2:active),
+  ::slotted(h3:active),
+  ::slotted(h4:active),
+  ::slotted(h5:active),
+  ::slotted(h6:active) {
+    border-color: var(--color-black-default);
+    background-color: transparent;
+  }
+
+  ::slotted(h1[active]),
+  ::slotted(h2[active]),
+  ::slotted(h3[active]),
+  ::slotted(h4[active]),
+  ::slotted(h5[active]),
+  ::slotted(h6[active]) {
+    border-color: var(--color-black-default);
+    background-color: var(--color-black-default);
+    color: var(--color-white-default);
+  }
+
+  ::slotted(h1[disabled]),
+  ::slotted(h2[disabled]),
+  ::slotted(h3[disabled]),
+  ::slotted(h4[disabled]),
+  ::slotted(h5[disabled]),
+  ::slotted(h6[disabled]) {
+    cursor: unset;
+    position: relative;
+    pointer-events: none;
+    background-color: var(--color-milk-default);
+    border-color: var(--color-milk-default);
+    color: var(--color-platinum-default);
+    --lyne-amount-color-override: var(--color-platinum-default);
+    //text-decoration: line-through;
+    &:after {
+      content: '';
+      position: absolute;
+      top: 50%;
```

Comment:
Ok in this way we have to write less code. I'm still thinking what could be the cleaner solution to center (vertically) the line, since it's 2px wide, it starts from the center and "grows" towards the bottom.

To center it vertically I used
`transform: translateY(-50%);`